typescript/packages/mesh-browser-lib/src/wrapper_conversion.ts

```diff
@@ -1,5 +1,6 @@
 import { SignedOrder } from '@0x/order-utils';
-import { BigNumber, providerUtils } from '@0x/utils';
+import { providerUtils } from '@0x/utils';
+import { BigNumber } from 'bignumber.js';
```

Comment:
This is an awful hack that I needed to implement so that the `BigNumber` types from `@0x/utils` and `@0x/types` don't conflict.